sample-cap/README.md

```diff
@@ -0,0 +1,85 @@
+# Sample CAP Application with SAP Cloud SDK for AI
+
+Sample CAP application written in TypeScript to demonstrate the usage of SAP Cloud SDK for AI.
+
+### Table of Contents
+
+- [Local Deployment](#local-deployment)
+- [Usage](#usage)
+  - [`ai-api`](#ai-api)
+    - [Deployment API](#deployment-api)
+  - [`foundation-models`](#foundation-models)
+    - [Azure OpenAI Chat Completion](#azure-openai-chat-completion)
+  - [`orchestration`](#orchestration)
+    - [Chat Completions with Templating](#chat-completions-with-templating)
+
+## Local Deployment
+
+1. Build the application with `pnpm install`.
+
+2. Login using `cf login -a API_ENDPOINT -o ORG -s SPACE`.
+
+3. Bind the application to your AI Core instance:
+
+   ```bash
+   cds bind -2 AI_CORE_INSTANCE_NAME
+   ```
+
+4. Run the application with the binding:
+
+   ```bash
+   cds bind -2 AI_CORE_INSTANCE_NAME --exec -- pnpm start
```

Comment:
after binding the instance in 3rd step is done, ```cds-tsx watch --profile hybrid``` should do it using the profile defined in the generated .cdsrc-private.json